src/file/path.rs

```diff
@@ -0,0 +1,160 @@
+use std::{fmt, path::PathBuf};
+
+/// [ssh_config(5)](https://linux.die.net/man/5/ssh_config) file is a sequence of entries, an entry is either a *keyword argument* line, or a *comment* line.
+///
+/// In contrary to man, we will be explicitly distinguishing comments and empty lines, though originally
+/// empty lines are considered comments.
+///
+/// This is an example of ssh_config file.
+///
+/// ```ssh-config
+/// # [% sshc 0.0.1 %]
+///
+/// Include dir/folder/file
+///
+/// # a simple entry
+/// Host example.com ssh.example.com
+///     Port 22
+///     User root
+///     Ciphers aes256-cbc,arcfour
+/// ```
+#[derive(PartialEq, Debug)]
+pub struct File {
```

Comment:
is it a good naming? is it a file? maybe it is a configuration? or a document?

Reply:
SSHConfigFile